.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,26 @@
+name: Agent Build CI
+
+on:
+  push:
+    paths:
+      - '**'
+      - '!docs/**'
+  pull_request:
+    paths:
+      - '**'
+      - '!docs/**'
+      
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
can you tell me the usage of this? We already have acaci.yml github workflow and the only difference I notice here is using actions/checkout@v2 instead of actions/checkout@v1 in acaci.yml.

Reply:
Just want to add CI to personal repository

Reply:
got it, so it will not be included in this PR before the final merging right?

Reply:
No. it will not be included.
How to remove it in PR?

Reply:
Hi @er1cthe0ne ,
Whether alcor dir need to be updated or not?
For this error : /mnt/host/code/src/../include/aca_util.h:95:71: error: 'NeighborType' is not a member of 'alcor::schema'

Reply:
> No. it will not be included.
> How to remove it in PR?

The instruction from this link should help: https://ffeathers.wordpress.com/2020/03/21/removing-an-updated-file-from-a-pr-on-github/

> Whether alcor dir need to be updated or not?
> For this error : /mnt/host/code/src/../include/aca_util.h:95:71: error: 'NeighborType' is not a member of 'alcor::schema'

yes, the alcor submodule needs to be updated because the NeighborType is a recent change. The alcor submodule index in master is already updated. To update the alcor submodule on your local clone, I suggest one of the below:
1. cd ~/dev/alcor-control-agent (or you local clone directory), run git submodule update --init --recursive
2. cd ~/dev/alcor-control-agent (or you local clone directory), cd alcor, run git pull
3. if the above doesn't help, cd ~/dev/alcor-control-agent (or you local clone directory), cd alcor, run git checkout master, but make sure not to include the alcor submodule change into your PR